src/sysupdate/sysupdate-resource.c

```diff
@@ -278,11 +280,13 @@ static int download_manifest(
         assert(ret_buffer);
         assert(ret_size);
 
+
         /* Download a SHA256SUMS file as manifest */
 
-        r = import_url_append_component(url, "SHA256SUMS", &suffixed_url);
+        r = import_url_append_component(url, manifest_file ?: "SHA256SUMS" , &suffixed_url);
+
```

Comment:
please remove this extra empty line.

(we generally try to keep function calls and their immediate return/error handling together, and insert empty lines *before* func calls or *after* error handling, but not between func and error handling.)